src/main/java/com/infoshareacademy/service/FridgeService.java

```diff
@@ -0,0 +1,48 @@
+package com.infoshareacademy.service;
+
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import com.infoshareacademy.fridge.Fridge;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Map;
+
+public class FridgeService {
+
+    public void showAllProductsInFridge(Map map) {
+        for (Object key : map.keySet()) {
+            System.out.println(key + " = " + map.get(key));
+        }
+    }
+
+/*    public void findInFridge(Fridge fridge, String search){
+
+
+        List<Fridge> findInFridge = list.stream().filter(list -> list.getProductInFridge().keySet().equals(search)).collect(Collectors.toList());
+
+
+    }*/
+
+
+    public void writeJson(Fridge fridge) throws IOException {
+        Path path = Path.of("src", "resources", "products_in_fridge.json");
+        File file = new File(path.toString());
+        ObjectMapper objectMapper = new ObjectMapper();
+        ObjectWriter objectWriter = objectMapper.writerWithDefaultPrettyPrinter();
+        objectWriter.writeValue(file, fridge);
+    }
+
+    public Map<String,Double> getJson() throws IOException {
+        Path path = Path.of("src", "resources", "products_in_fridge.json");
+        ObjectMapper objectMapper = new ObjectMapper();
+        File file = new File(path.toString());
+        Map<String, Double> fridge = objectMapper.readValue(file, new TypeReference<Map<String, Double>>(){});
+        return fridge;
+    }
```

Comment:
Duplikacja kodu jak w klasie RecipeService. Warto pomyśleć, aby na przyszłość to zmienić.